controllers/feeds/getFeeds.js

```diff
@@ -16,16 +16,16 @@ const {
   isPostBlocked,
   modifyReactionsPost
 } = require('../../utils/post');
-const {DomainPage, Locations, User} = require('../../databases/models');
+const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
+const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports `UserFollowUser` and `UserFollowUserFunction` are added correctly. However, it's important to ensure that the `UserFollowUserFunction` module exports a function named `checkIsBlurredPost`, as it is used later in the code.

```javascript
const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->